src/notation/qml/MuseScore/NotationScene/ScorePropertiesDialog.qml

```diff
@@ -0,0 +1,220 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 1.4
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.NotationScene 1.0
+
+StyledDialogView {
+    id: root
+
+    title: qsTrc("notation", "Score properties")
+
+    contentWidth: 682
+    contentHeight: 406
+
+    margins: 15
+
+    ScorePropertiesModel {
+        id: scorePropertiesModel
+    }
+
+    Component.onCompleted: {
+        scorePropertiesModel.load()
+    }
+
+    ColumnLayout {
+        anchors.fill: parent
+        width: parent.width
+        spacing: 8
+        Column {
+            id: headerPart
+            spacing: 5
+            readonly property var headerTags: scorePropertiesModel.headerTags
+            RowLayout {
+                width: root.contentWidth
+                spacing: 10
+                StyledTextLabel {
+                    Layout.preferredWidth: 110
+                    height: 30
+                    text: qsTrc("notation", "File path:")
+                    font: ui.theme.bodyBoldFont
+                    horizontalAlignment: Text.left
+                }
+
+                TextInputField {
+                    Layout.fillWidth: true
+                    height: 30
+                    currentText: headerPart.headerTags[0]
+                    readOnly: true
+                }
+
+                FlatButton {
+                    width: 30
+                    height: 30
+                    icon: IconCode.OPEN_FILE
+                    onClicked: scorePropertiesModel.openFileLocation()
+                }
+            }
+
+            RowLayout {
+                width: root.contentWidth
+                height: 30
+                spacing: 10
+                StyledTextLabel {
+                    Layout.preferredWidth: 110
+                    text: qsTrc("notation", "MuseScore version: ")
```

Comment:
It is preferred to exclude the trailing space from the translatable string, since translators might miss it. I don't think a trailing space is necessary anyway in this case, since the RowLayout already has spacing. 

Also, I would change that spacing from 10 to 12, to be consistent with the rest of the app (we seem to prefer multiples of 4).

Reply:
Oh okay. I had initially used the default width, so I needed to give some space.